settings.py

```diff
@@ -572,6 +572,7 @@
 CELERY_PERIODIC_QUEUE = 'celery_periodic'
 CELERY_REMINDER_RULE_QUEUE = 'reminder_rule_queue'
 CELERY_REMINDER_CASE_UPDATE_QUEUE = 'reminder_case_update_queue'
+CELERY_REMINDER_CASE_UPDATE_BULK_QUEUE = 'reminder_case_update_bulk_queue'
```

Comment:
we could default this to the same as `CELERY_REMINDER_CASE_UPDATE_QUEUE` and then only override in localsettings on envs that need it. That would make rollout easier.

Reply:
5390b9efe9d3a1fc460ac6077c84b6aaf7751458